packages/react/src/codemods/emotion-to-compiled/emotion-to-compiled.tsx

```diff
@@ -116,76 +83,120 @@ const replaceEmotionCoreCSSTaggedTemplateExpression = (
-  });
+const handleClassNamesCXBehavior = (j: core.JSCodeshift, objectPattern: ObjectPattern) => {
+  const cxIdentifierName = 'cx';
+  const axIdentifierName = 'ax';
+  const compiledRuntimePackageName = `${COMPILED_IMPORT_PATH}/runtime`;
+
+  const cxObjectPropertyCollection = j(objectPattern)
+    .find(j.ObjectProperty)
+    .filter(
+      (objectPropertyPath) =>
+        objectPropertyPath.node.key.type === 'Identifier' &&
+        objectPropertyPath.node.key.name === cxIdentifierName
+    );
+
+  cxObjectPropertyCollection.forEach((cxObjectPropertyPath) => {
+    addCommentBefore({
+      j,
+      collection: j(cxObjectPropertyPath.node),
+      message: `Please replace "${cxIdentifierName}" with "${axIdentifierName}" from "${compiledRuntimePackageName}".
+      Usage: import { ${axIdentifierName} } from '${compiledRuntimePackageName}';
+
+      NOTE: Both "${cxIdentifierName}" and "${axIdentifierName}" have some differences, so we have not replaced its usage.
+      Please check the docs for "${axIdentifierName}" usage.
+
+      In future, we will expose "${axIdentifierName}" directly from "${imports.emotionCoreImportNames.ClassNames}" props.
 
-  const emotionCoreImportValues = Object.values(imports.emotionCoreImportNames);
+      Issue tracked on Github: https://github.com/atlassian-labs/compiled/issues/373`,
+    });
+  });
+};
 
-  importSpecifiers
-    .filter((identifierPath) => !emotionCoreImportValues.includes(identifierPath.name))
-    .forEach((importSpecifierPath) => {
-      collection.find(j.Identifier).some((identifierPath) => {
-        const name = identifierPath.node.name;
+const handleClassNamesStyleBehavior = (j: core.JSCodeshift, objectPattern: ObjectPattern) => {
+  const styleIdentifierName = 'style';
 
-        const isValidIdentiferFound = name === importSpecifierPath.name;
+  const hasStyleObjectProperty = j(objectPattern)
+    .find(j.ObjectProperty)
+    .some(
+      (objectPropertyPath) =>
+        objectPropertyPath.node.key.type === 'Identifier' &&
+        objectPropertyPath.node.key.name === styleIdentifierName
+    );
 
-        if (isValidIdentiferFound) {
-          addCommentToStartOfFile({
-            j,
-            collection,
-            message: `
-              "${name}" is not exported from "${imports.compiledPackageName}" at the moment. Please find an alternative for it.
-            `,
-          });
+  if (!hasStyleObjectProperty) {
+    const styleObjectProperty = j.objectProperty(
+      j.identifier(styleIdentifierName),
+      j.identifier(styleIdentifierName)
+    );
 
-          return true;
-        }
+    objectPattern.properties.push(styleObjectProperty);
 
-        return false;
-      });
+    addCommentBefore({
+      j,
+      collection: j(styleObjectProperty),
+      message: `We have exported "${styleIdentifierName}" from "${imports.emotionCoreImportNames.ClassNames}" props.
+      If you are using dynamic declarations, make sure to set the "${styleIdentifierName}"
```

Comment:
would be good to link to the website docs here https://compiledcssinjs.com/docs/class-names#dynamic-declarations

Reply:
I wanted to. But then I thought, its old website.

Reply:
Ahh..you took url from preview one.

Reply:
omg you're up late!!

Reply:
Going out soon. hehe. Thought of wrapping up this PR.